src/components/badgeable/__tests__/Badgeable-test.js

```diff
@@ -4,12 +4,37 @@ import Badgeable from '../Badgeable';
 
 describe('components/badgeable/Badgeable', () => {
     test('should correctly render children in a badgeable wrapper without badges', () => {
-        const children = 'some text to render';
+        const children = <div id="render">Some text to render</div>;
         const wrapper = shallow(<Badgeable>{children}</Badgeable>);
 
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should pass down props to children when it is an element', () => {
+        const wrapper = shallow(
+            <Badgeable foo="bar">
+                <div id="render">Some text to render</div>
+            </Badgeable>,
+        );
+        expect(
+            wrapper
+                .find('#render')
+                .first()
+                .prop('foo'),
+        ).toEqual('bar');
+    });
+
+    test('should throw an error when there are multiple children', () => {
```

Comment:
The component is losing the ability to render text and multiple components, which seems like a breaking change? Does these features get used anywhere internally or externally?